packages/@okta/vuepress-theme-prose/components/Search.redesign.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div class="search--wrapper" data-search-bar>
+    <div class="SearchBox" data-pipeline="developer-okta-com">
+      <div class="CoveoOmnibox"></div>
+      <div class="CoveoAnalytics"></div>
+    </div>
+    <img class="input--icon" src="/img/icons/search.svg" />
+  </div>
+</template>
+
+<script>
+export default {
+  mounted() {
+    // Mount Coveo after page load
```

Comment:
Ah, is this why I see the search seem to appear a moment after the rest of the page renders?

Reply:
yup